src/NHibernate.TestDatabaseSetup/TestDatabaseSetup.cs

```diff
@@ -19,15 +21,17 @@ public class DatabaseSetup
 			{
 				{"NHibernate.Driver.SqlClientDriver", SetupSqlServer},
 				{"NHibernate.Driver.Sql2008ClientDriver", SetupSqlServer},
-				{"NHibernate.Driver.OdbcDriver", SetupSqlServerOdbc},
 				{"NHibernate.Driver.FirebirdClientDriver", SetupFirebird},
-				{"NHibernate.Driver.SQLite20Driver", SetupSQLite},
 				{"NHibernate.Driver.NpgsqlDriver", SetupNpgsql},
 				{"NHibernate.Driver.OracleDataClientDriver", SetupOracle},
 				{"NHibernate.Driver.MySqlDataDriver", SetupMySql},
 				{"NHibernate.Driver.OracleClientDriver", SetupOracle},
 				{"NHibernate.Driver.OracleManagedDataClientDriver", SetupOracle},
+#if NET461
+				{"NHibernate.Driver.OdbcDriver", SetupSqlServerOdbc},
```

Comment:
Odbc is supported, and I have tested its setup locally with Core target, it works.